[2022-08-19 23:09:01,964] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:54:00+00:00 [queued]>
[2022-08-19 23:09:01,985] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:54:00+00:00 [queued]>
[2022-08-19 23:09:01,986] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:09:01,987] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-19 23:09:01,988] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-19 23:09:02,026] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_telegram_version> on 2020-01-01 03:54:00+00:00
[2022-08-19 23:09:02,040] {standard_task_runner.py:52} INFO - Started process 2887 to run task
[2022-08-19 23:09:02,073] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'sefa_dag', 'get_telegram_version', 'scheduled__2020-01-01T03:54:00+00:00', '--job-id', '489', '--raw', '--subdir', 'DAGS_FOLDER/sefa_dag.py', '--cfg-path', '/tmp/tmpy5kkx6f6', '--error-file', '/tmp/tmpo4arv1ap']
[2022-08-19 23:09:02,084] {standard_task_runner.py:80} INFO - Job 489: Subtask get_telegram_version
[2022-08-19 23:09:02,232] {task_command.py:371} INFO - Running <TaskInstance: sefa_dag.get_telegram_version scheduled__2020-01-01T03:54:00+00:00 [running]> on host fd7a784b9251
[2022-08-19 23:09:02,381] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=sefa_dag
AIRFLOW_CTX_TASK_ID=get_telegram_version
AIRFLOW_CTX_EXECUTION_DATE=2020-01-01T03:54:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-01T03:54:00+00:00
[2022-08-19 23:09:02,383] {python.py:173} INFO - Done. Returned value was: 1.24.0
[2022-08-19 23:09:02,406] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=sefa_dag, task_id=get_telegram_version, execution_date=20200101T035400, start_date=20220819T230901, end_date=20220819T230902
[2022-08-19 23:09:02,473] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-19 23:09:02,530] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
